environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -203,10 +211,23 @@
         disk_format: qcow2
         filename: /tmp/ubuntu.img
         min_disk: 3
+        min_ram: 512
         properties:
           cpu_arch: x86_64
           distro: ubuntu
           hw_rng_model: virtio
+          architecture: x86_64
+          hypervisor_type: qemu
```

Comment:
Shouldn't this be kvm by default? If available, I would want to use hardware virtualization (nested if needed). Doesn't kvm fall back to qemu if no HW support is available?